Tests/SourceControlTests/CheckoutManagerTests.swift

```diff
@@ -23,18 +23,28 @@ private enum DummyError: Swift.Error {
 
 private class DummyRepository: Repository {
     var tags: [String] = ["1.0.0"]
+    unowned let provider: DummyRepositoryProvider
+
+    init(provider: DummyRepositoryProvider) {
+        self.provider = provider
+    }
 
     func resolveRevision(tag: String) throws -> Revision {
         fatalError("unexpected API call")
     }
 
+    func fetch() throws {
+        provider.numRepoFetches += 1
+    }
+
     func openFileView(revision: Revision) throws -> FileSystem {
         fatalError("unexpected API call")
     }
 }
 
 private class DummyRepositoryProvider: RepositoryProvider {
     var numFetches = 0
+    var numRepoFetches = 0
```

Comment:
Hm do you have a suggestion? 
numFetches -> numRepoClones, numBareFetches?
numRepoFetches -> numRepoUpdates?